php/Dockerfile

```diff
@@ -7,7 +7,7 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && \
     apt-get --no-install-recommends --no-install-suggests --yes --quiet install \
         apt-transport-https bash-completion ca-certificates curl git gnupg imagemagick \
-        less mongodb-clients mysql-client perceptualdiff ssh-client sudo vim wget && \
+        less mongodb-clients mysql-client perceptualdiff ps ssh-client sudo vim wget && \
```

Comment:
why do you have added PS, WGET and SSH-CLIENT?
PS is accessible on docker ps command on specific container. And VIM can't be installed on image for many decurity reason. a container can be omnipotent.